catalog/USGS/USGS_NLCD_RELEASES_2019_REL_RCMAP_V5_TRENDS_YEAR.jsonnet

```diff
@@ -0,0 +1,441 @@
+        'gee:units': units.count,
+      },
+      {
+        name: 'non_sagebrush_shrub_break_point',
+        description: 'Presence or absence of structural breaks in each component for each year.',
+        'gee:units': units.count,
+      },
+      {
+        name: 'perennial_herbaceous_break_point',
+        description: 'Presence or absence of structural breaks in each component for each year.',
+        'gee:units': units.count,
+      },
+      {
+        name: 'tree_break_point',
+        description: 'Presence or absence of structural breaks in each component for each year.',
+        'gee:units': units.count,
+      },
+      {
+        name: 'annual_herbaceous_segment_pvalue',
+        description: 'P-value of structural breaks segment in each component for each year.',
+        'gee:scale': 0.01,
+        'gee:units': units.percent,
+      },
+      {
+        name: 'bare_ground_segment_pvalue',
+        description: 'P-value of structural breaks segment in each component for each year.',
+        'gee:scale': 0.01,
+        'gee:units': units.percent,
+      },
+      {
+        name: 'herbaceous_segment_pvalue',
+        description: 'P-value of structural breaks segment in each component for each year.',
+        'gee:scale': 0.01,
+        'gee:units': units.percent,
+      },
+      {
+        name: 'litter_segment_pvalue',
+        description: 'P-value of structural breaks segment in each component for each year.',
+        'gee:scale': 0.01,
+        'gee:units': units.percent,
+      },
+      {
+        name: 'sagebrush_segment_pvalue',
+        description: 'P-value of structural breaks segment in each component for each year.',
+        'gee:scale': 0.01,
+        'gee:units': units.percent,
+      },
+      {
+        name: 'shrub_segment_pvalue',
+        description: 'P-value of structural breaks segment in each component for each year.',
+        'gee:scale': 0.01,
+        'gee:units': units.percent,
+      },
+      {
+        name: 'non_sagebrush_shrub_segment_pvalue',
+        description: 'P-value of structural breaks segment in each component for each year.',
+        'gee:scale': 0.01,
+        'gee:units': units.percent,
+      },
+      {
+        name: 'perennial_herbaceous_segment_pvalue',
+        description: 'P-value of structural breaks segment in each component for each year.',
+        'gee:scale': 0.01,
+        'gee:units': units.percent,
+      },
+      {
+        name: 'tree_segment_pvalue',
+        description: 'P-value of structural breaks segment in each component for each year.',
+        'gee:scale': 0.01,
+        'gee:units': units.percent,
+      },
+      {
+        name: 'annual_herbaceous_segment_slope',
+        description: 'Slope of structural breaks segment, given in units of % change/year, in each component for each year.',
+        'gee:scale': 0.01,
+        'gee:units': units.percent,

```

Comment:
I think we may want a new unit for this, since the slope represents percent per year. Feel free to add TODO(dpencosk): Add units.percent_per_year.

Reply:
Gotcha, yeah, this one was a little bit tricky. Adding a new unit for this would be great since it will come up again when the V6 comes out. Thanks!

Reply:
Updated (added the TODO line to the first segment_slope, but new unit would apply to all segment_slope bands. Thanks